dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageChecker.java

```diff
@@ -239,8 +239,14 @@ private boolean problemFilter(LinkageProblem linkageProblem) {
     String targetClassName = symbol.getClassBinaryName();
     String methodName = symbol.getName();
 
-    // Skip references to Java runtime class. For example, java.lang.String.
-    if (classDumper.isSystemClass(targetClassName)) {
```

Comment:
This was hiding the protobuf's linkage error against Java 8 runtime https://github.com/protocolbuffers/protobuf/issues/7827